gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=permwrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-bin.zip
```

Comment:
what's the reason for switching gradle vesions?

Reply:
I was originally testing with GradleRIO 2022 which used 7.x, and then went back to 2021 which used 6.0, I was just matching its version.